packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -65,25 +66,26 @@ export const isNativeEvmAsset = (assetId: AssetId): boolean => {
   }
 }
 
+// converts an asset to zrx token (symbol or contract address)
 export const assetToToken = (asset: Asset): string => {
   const { assetReference, assetNamespace } = fromAssetId(asset.assetId)
   return assetNamespace === 'slip44' ? asset.symbol : assetReference
 }
 
 export const getUsdRate = async (sellAsset: Asset): Promise<string> => {
```

Comment:
We might want to call this `getUsdcRate` - calling a quote `usdRate` is effectively semantically equal to hardcoding USDC to 1 (which we're doing on l.80). Won't change the actual harcoding logic, but will make it easier to tackle it to be more accurate if we want to move away from the hardcoding eventually

Reply:
This was just moved, I can change this function, but swapper api will still be `getUsdRate`. Not sure it's worth messing with it at this time?

Reply:
Yeah totally fine to tackle it at a later time! Just something to keep in mind